spacy/tests/test_misc.py

```diff
@@ -139,7 +139,13 @@ def test_load_model_blank_shortcut():
     nlp = util.load_model("blank:en")
     assert nlp.lang == "en"
     assert nlp.pipeline == []
+
+    # ImportError for loading an unsupported language
     with pytest.raises(ImportError):
+        util.load_model("blank:zxx")
+
+    # ValueError for requesting an invalid language code that isn't registered
+    with pytest.raises(ValueError):
         util.load_model("blank:fjsfijsdof")
```

Comment:
Hm actually now that I see it like this, maybe this is confusing behaviour. A user might not necessarily understand why some invalid codes are giving an `ImportError` and others a `ValueError`. Instead, perhaps we can just `pass` when we catch the `LanguageTagError` and then the fall-back `else` will raise the same `ImportError` as before?